apps/lobe-chat/1.35.11/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   lobe-chat:
-    image: lobehub/lobe-chat:v1.35.11
+    image: lobehub/lobe-chat:v1.35.12
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The primary differences between the two versions (v1.35.11 and v1.35.12) of the code are:

- Version 1.35.11 has been updated to version 1.35.12.

This change suggests that there have been some improvements or updates to LobeChat over time. It may be helpful to review why this update was necessary based on any relevant documentation or considerations in the project description. If you need a detailed technical analysis of specific aspects of the code or the changes made recently, please provide more information about the specifics you'd like reviewed.